### ISSUE TITLE
Develop community section on Homepage

### ISSUE THREAD

**Issue by jorgemgoncalves** [snowplow/snowplowanalytics.com#290]

**Comment by yalisassoon** [comment]
Inspiration: mixpanel.com

**Comment by yalisassoon** [comment]
Would love an update on this @jorgemgoncalves !

**Comment by yalisassoon** [comment]
Haven't made progress yet - still pending

**Comment by jorgemgoncalves** [comment]
Hi @yalisassoon and @Mandelliant, we started to work on the design of this section and gathered quite a few ideas but we got to a point where we need your opinion to move on.

Basically, we imagine this block showing the next meet-up date, having a snippet of your last post, to show the latest or trending thread on Discourse. The problem is that this block would be much more dynamic than your Community.

So we can have two approaches...either we go full dynamic and also change the Community page or we choose a more static approach and make a less dynamic block.

What do you feel should be done?

Here are 2 experiences that we've been working on but don't be influenced by the design it is still in the making.

[Version 1](https://projects.invisionapp.com/share/WRGCWSUYT9A#/screens/285287602_Snowplow_Website_Homepage_Comunity_1)
[Version 2](https://invis.io/WRGCWSUYT9A#/285289013_Snowplow_Website_Homepage_Comunity_1_V1)

**Comment by yalisassoon** [comment]
Hi @jorgemgoncalves 

Of the two designs I prefer the colourful one:

![image](https://user-images.githubusercontent.com/201147/37516952-d0a65bae-2907-11e8-896f-578dc74a8d57.png)

However, neither enable us to surface specific content on the homagepage e.g. specific white papers, or up and coming meetups. We'll want to curate these. So when we launch e.g an eBook on product analytics, we'll want to list this under "white papers and resources'. The problem with the proposed approach is they're not compelling calls to action. Why should anyone click on our Github? Or our blog? They might click on an interesting thought piece (which happens to be on our Discourse). Or they might be excited about a specific meetup we're advertising.

That means a couple of things:

1. We need a new headings. @Mandelliant not sure what you reckon but I think these should be:
  - eBooks, white papers and thought leadership
  - Debate and discussion
  - News
  - Events
2. We need to be able to surface individual items in each section on the homepage - like Mixpanel has done.
3. We need a way in Jekyll to easily be able to change what content is listed

Does that make sense?

**Comment by jorgemgoncalves** [comment]
Hi @yalisassoon and @Mandelliant, 

First of all, I need to apologise for not explaining the preview flow. If you press the plus button on Snowplow Blog you will see the continuation window of what would happen. 

You are totally right! It is very important to have more compelling information and even visual cues that make users interact, that is why, on the second test, we included more visual alerts, like the little red circle notifications (something that we're all used to on apps) to inform visitors of new information.

One of our main conclusions is that each block needs to have its own behaviour and size (announcing a meeting requires less space than a placing a blog post snippet).

Our idea is to create these blocks totally automated or the most automated possible, requiring just a little update on a single data file for instance. 

What is your opinion @Mandelliant?

**Comment by yalisassoon** [comment]
Ah! That makes more sense!

I guess my main concern is that we're putting that interesting content "one click away" when maybe we could preview at least some of it immediately on homepage load?

@Mandelliant what do you reckon?

**Comment by Mandelliant** [comment]
@yalisassoon I agree with your suggestions for headers. I like the colorful one as well and I see what you mean about keeping content "one click away." 

What if we had the first section expanded on page load (like the picture), and then cycle through each section as if the user was navigating? We could display a sample of the most recent content for each category like Jorge has now for the blog. @jorgemgoncalves is it possible to include images, too?

Does that all make sense?

<img width="1205" alt="screen shot 2018-03-16 at 12 31 56 pm" src="https://user-images.githubusercontent.com/8901138/37532731-39b4552a-2916-11e8-8196-81ced88d3b1f.png">

**Comment by jorgemgoncalves** [comment]
Hi @Mandelliant & @yalisassoon, I've just pushed the final version of the Community Section to QA.

Can you please check and give me feedback so I can create the PR to next?

**Comment by jorgemgoncalves** [comment]
Hi @Mandelliant and @yalisassoon, I've just created the PR of the Community Section to next.snowplowanalytics.com.

The copy that is in the module isn't final. If you have any questions don't hesitate to ask.

**Comment by yalisassoon** [comment]
@Mandelliant - do you want to have a first crack:

1. Deciding what content we want to put on
2. Test and put live
